### ISSUE TITLE
Make it harder to progress with dragons.

### ISSUE THREAD

**Issue by zucaluca** [AlexModGuy/Ice-and-Fire-Suggestions#374]
I feel once you hatch your own dragon, you are set on conquering all other mobs. 

These ideas can be built upon or transformed into something else, the purpose is to make it more difficult for dragon stage progression.

1. Having a dragon egg only start the hatching process (with the physical conditions already implemented) if the player has placed more than 1 fire/frost flower around it. This means a dragon would have to hatch in the specific area that can hold those breeding flowers.

2. Once the dragon hatches, up until stage 3, every full moon it needs to go on a killing spree of about 10 vanilla animals. If it doesnt, it may run away and never return, evaporate into thin air, or attempt to kill the player. This may force the player to make a semi-large farm before or soon after  hatching as well as watch the moon stages. If a player sleeps before a full moon gets to midnight, it skips this process and the dragon waits until next full moon. If the dragon is in a horn, it will come out at full moon unless the player sleeps. This may prevent the player from going underground without a clock and bed, or until the dragon is after stage 3.

3. For a dragon to transition to a stage 5, it needs to have gotten an achievement of killing a gorgon, cyclops, another stage 4 dragon, or another difficult to kill ice and fire mob. If the dragon kills a mob after it is ready to transition, it will soon after become a stage 5. If the player uses dragon meal to increase growth at any stage, it will work except for the stage 5 killing requirements.

4. If the dragon wants to become stage 6 at the maximum amount of days, it must defeat a bunch of lesser staged dragons (stage doesnt matter,). It must be in kill mode, though. <-- More on that later. This will brand the dragon as a beast that kills its own kind and the dragon will be shaded much darker and grow a tad bit bigger and is known to leave the player every couple days and bring back dragon bones, scales, etc that it drops for the player. Of course, you dont need to make it stage 6 if killing other dragons isnt your thing. So the killing process for turning into stage 6 can be configured, you can right click the dragon with an item that toggles kill mode or regular mode. Kill mode of course turning it into stage 6 when killing enough dragons in this mode. This mode gives the dragon dangerous looking eyes, while regular mode is regular stage 5 mode. The issue with a stage 6 dragon, though, is that a random seige of ice and fire mobs will attack at rare intervals and try to defeat your dragon. This may make a large mess around your base, so it is best to keep the dragon on lockdown so it cant see the mobs and the mobs cant see it. This may make the mobs stationary (because they have just spawned near the dragon but cannot see it. Now you can farm those attackers for loot, or have your other stage 5 kill mode'd dragons kill the dragons in those raids to easily level up other stage 5 dragons you own into stage 6. Turning a dragon into stage 6 is quite dangerous because the raids against it may decide to kill your farm animals if they arent protected.
It may be very difficult for stage 6 dragons to breed with the rare attacks happening or the dragon flying off to bring back loot. This dragon is hard to control as well when flying it. It may decide to swoop down and kill a random mob. All of these stage 6 atrocities can be dealt with though.

4a. To control the stage 6 dragon easier when flying or even toggling its random lootings (which will also turn off killings attempted against the dragon), you have to make it respect you. This means unsitting it and attacking it until it has a very low amount of health, and right clicking it with your own sword. It will then stand up and be as tame as a stage 5, with the added growth and possibly strength. So yes, you could completely avoid everything it does as a stage 6 and just defeat it so it respects you. To switch back to looting and pillaging mode so it brings back loot, raids against it, and it swooping down on mobs with you on its back, all you need to do is hit it from behind with a wooden stick/sword. This will show cowardice and you will lose the dragons respect. To go back to respect mode yes, you must repeat the respect process. I know this is kinda dark, which is why it is recommended not to go to stage 6 for younger kids. This could be an addon mod for people who want stage 6.
Feel free to build off these ideas or make them less dark.

This was a rambling so it may be as confusing to you as it was to me.

Too long, didn't read.

**Comment by taisong0612** [comment]
So how are you going to prepare all the sacrifice dragons?
Am I supposed to kill my other dragons that I tamed?

**Comment by zucaluca** [comment]
Yes. I was thinking you can mass breed them with your stage 5 female and male.

**Comment by zucaluca** [comment]
You can, or if there are stage 3 roosts nearby you can those as well. Once you unlock the pillagings for the stage 6 you can just farm the dragons off those pillagings to get more stage 6 of your own. 

If there arent any more dragons nearby (multiplayer server), then you will have to kill tamed dragons.
